atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/AbstractDegradedClusterTest.java

```diff
@@ -58,10 +58,21 @@ public abstract class AbstractDegradedClusterTest {
             new HashMap<>();
 
     private String schemaAtStart;
+    private final boolean withSchemaVersionRecording;
+
+    public AbstractDegradedClusterTest(boolean withSchemaVersionRecording) {
+        this.withSchemaVersionRecording = withSchemaVersionRecording;
+    }
+
+    public AbstractDegradedClusterTest() {
+        this(true);
+    }
 
     @Before
     public void recordSchemaVersion() throws TException {
-        schemaAtStart = getUniqueReachableSchemaVersionOrThrow();
+        if (withSchemaVersionRecording) {
+            schemaAtStart = getUniqueReachableSchemaVersionOrThrow();
+        }
```

Comment:
This request causes our Availability test to fail (since it throws an exception). Note that `schemaAtStart` is never used by these tests anyway, so we can ignore it for AvailabilityTests.